src/SfxWeb/src/app/views/application/events/events.component.ts

```diff
@@ -11,7 +11,7 @@ import { IEventStoreData } from 'src/app/modules/event-store/event-store/event-s
 })
 export class EventsComponent extends ApplicationBaseControllerDirective {
 
-  listEventStoreData: IEventStoreData [];
+  listEventStoreData: IEventStoreData<any, any> [];
```

Comment:
This is just a workaround, once it its approved the necessary changes will be made.